README.md

```diff
@@ -16,6 +16,14 @@ I recommend creating a separate package inside your project to serve as the cont
 
 For an example of how to use the resulting package, check out `example/example.go`. You can also see the API it generates at [godoc.org](https://godoc.org/github.com/bouk/staticfiles/files).
 
+## Installation
+
+Install with
+
+```
+go get github.com/bouk/staticfiles
```

Comment:
I don't think -u should be added here by default.

Those who know what they're doing and want to update their library to latest available version can run it with -u flag.